extensions/2.0/Vendor/MPEG_accessor_timed/README.md

```diff
@@ -0,0 +1,98 @@
+|**suggestedUpdateRate**|`number`|This property provides the frequency at which the renderer is recommended to poll the underlying buffer for new data.|No, default: `25`|
+|**extensions**|`object`|JSON object with extension-specific objects.|No|
+|**extras**|[`any`](#reference-any)|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [MPEG_accessor_timed.schema.json](/Extensions/MPEG_accessor_timed/schema/MPEG_accessor_timed.schema.json)
+
+### MPEG_accessor_timed.immutable
+
+This flag equal to false indicates the accessor information `componentType`, `type`, and `normalized` may change over time. The changing values of `componentType`, `type` and `normalized` are provided through accessor information header. This flag equal to true indicates the accessor information `componentType`, `type`, and `normalized` do not change over time and are not present in the accessor information header. Note that count, max, min, and byteOffset are expected to change and are always included as part of the timed accessor information header.
+
+* **Type**: `boolean`
+* **Required**: No, default: `true`
+
+### MPEG_accessor_timed.bufferView
+
+This property provides the index in the `bufferViews` array to a `bufferView` element that points to the timed accessor information header as described in ISO/IEC 23090-14:Table 8. byteLength field of the `bufferView` element indicates the size of the timed accessor information header. The buffer properties in the `bufferView` element shall point to the same buffer as the `bufferView` in the containing accessor object. In the absence of the `bufferView` attribute, it shall be assumed that the buffer has no dynamic header. In that case, the immutable flag shall be present and shall be set to true.
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+### MPEG_accessor_timed.suggestedUpdateRate
+
+The suggestedUpdateRate provides the frequency at which the renderer is recommended to poll the underlying buffer for new data. The rate is provided in number of changes per second.
+
+* **Type**: `number`
+* **Required**: No, default: `25`
+* **Minimum**: ` >= 0`
+
+### MPEG_accessor_timed.extensions
+
+JSON object with extension-specific objects.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: Extension
+
+### MPEG_accessor_timed.extras
+
+Application-specific data.
+
+* **Type**: [`any`](#reference-any)
+* **Required**: No
+
+
+## Known Implementations
+
+* [ISO/IEC WD 23090-24](https://www.iso.org/standard/83696.html)
+
+## Resources
+
+* [ISO/IEC FDIS 23090-14](https://www.iso.org/standard/80900.html), Information technology — Coded representation of immersive media — Part 14: Scene Description 
+* [ISO/IEC WD 23090-24](https://www.iso.org/standard/83696.html), Information technology — Coded representation of immersive media — Part 24: Conformance and Reference Software for Scene Description for MPEG Media
+
+## License
+
+Copyright (c), ISO/IEC
```

Comment:
This line should include year, also (c) is redundant. Consider changing it to:
> Copyright ISO/IEC 2022

Reply:
Implemented.